### PR TITLE
Added French to multi-language

### PR DESCRIPTION
**Pull request by Tissy**
Being half French, I thought I would add my language translation contribution :)

### CODE REVIEW

**Comment by gabrielcox** [comment]
Wow! 2 languages within hours of each other.  Is your PR with consideration that the Japanese PR with come through first?  That would be ideal to minimize conflicts.  It looks like both have similar changes to strings.xml.  If you haven't already, please have a look at and analyze the Japanese PR to see how it may impact yours.

We really appreciate your contribution.  This is great!

**Comment by friissoren** [comment]
Thanks. I am currently traveling. Please wait a few days and I can take a look at this. We need to figure out how to deal with the conflicts between this PR and PR #58.

**Comment by Tissy** [comment]
I did it off the back of the Japanese translation.  Wonder if there is a simple way to perhaps add a menu function to select the required language?

**Comment by gabrielcox** [comment]
I think the Main Android Settings/Language settings can change it back and forth for testing.   As I understand it, based on your locale setting, within the App, Android will use the language your device is set to if the language strings are there (as the ja and fr PRs add).  The the language file is not there, it will use the "default" strings.xml.

File: Android/app/src/main/res/values_fr/strings.xml [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    <string name="coded_phy_supported">Coded Phy supporté</string>
+    <string name="ea_not_supported">Publicité etendue non supporté</string>
+    <string name="ea_supported">Publicité etendue supporté</string>
+    <string name="nan_not_supported">WiFi NaN non supporté</string>
+    <string name="nan_supported">WiFi NaN supporté</string>
+
+    <string name="wifi_beacon_scan_not_supported">Balayage des balise WiFi non supporté</string>
+    <string name="wifi_beacon_scan_supported">Balayage des balise WiFi supporté</string>
+
+    <string name="permission_rationale_location">Cette application ne fonctionnera pas sans activer l'accès à la localisation</string>
```
Comment (friissoren) [inline Android/app/src/main/res/values_fr/strings.xml:31]: Please add a "\\" before the "\'" character.
l\\'accès

File: Android/app/src/main/res/values_fr/strings.xml [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    <string name="app_name">OpenDroneID</string>
+    <string name="bt_not_supported">Bluetooth non pris en charge</string>
+    <string name="bt_not_enabled_leaving">Bluetooth non activé, laissant</string>
+    <string name="wifi_not_enabled_leaving">WiFi non activé, laissant</string>
+    <string name="info">Information</string>
+
+    <string name="Clear">Dégager</string>
+    <string name="Help">Aider</string>
+    <string name="Log_enabled">Journal Activé</string>
+    <string name="Show_log_location">Afficher l'emplacement du journal</string>
```
Comment (friissoren) [inline Android/app/src/main/res/values_fr/strings.xml:19]: Please add a "\\" before the "\'" character.
l\\'emplacement

File: Android/app/src/main/res/values_fr/strings.xml [hunk 3]
```diff
@@ -0,0 +1,88 @@
+<!--
```
Comment (friissoren) [inline Android/app/src/main/res/values_fr/strings.xml:1]: The directory path needs to be fixed so Android Studio finds this file. Please change values_fr to values-fr.

**Review by friissoren — CHANGES_REQUESTED** [review]
I very much appreciate your contribution. Having more languages supported in the app is a great idea.

However, you clearly didn't compile this :-)

When you fix these few items, could you also rebase the patches on top of the latest master (which now have the Japanese translations) and then squash the two resulting commits? Thanks.

**Comment by friissoren** [comment]
BTW: I tested Gabriel's suspicion that the app language will change with the main phone system language. I tried this on a virtual/emulator device in Android studio. When you change the system language in the settings menu, even an open instance of the app will change the displayed language, when you switch over to it. Very nice.

**Comment by friissoren** [comment]
Thanks. I have merged this now.

Some of those translations look rather long. You might at some point try to fire up the app and get a transmitter visible in the list, open the Info view and then check the layout of the detailed info view. That view is quite "busy" and might wrap the text strangely when the fields become too long.

**Comment by Tissy** [comment]
Yup, totally agree.  I shall make some amendments :)

**Pull request merged**